src/Observable.php

```diff
@@ -1473,7 +1553,7 @@ public function repeatWhen(callable $notifier)
      * @param SchedulerInterface $scheduler
      * @return AnonymousObservable
      */
-    public function subscribeOn(SchedulerInterface $scheduler)
+    public function subscribeOn(SchedulerInterface $scheduler): AnonymousObservable
```

Comment:
We don't have observeOn.  Now that the scheduler isn't being passed through subscribe, I don't know if it does anything useful.